modules/styling/stories/examples/CSProp.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import {createStencil, cssVar} from '@workday/canvas-kit-styling';
+import {Card} from '@workday/canvas-kit-react/card';
+
+const myStencil = createStencil({
+  vars: {
+    color: '#E0AAFF',
+  },
+  base: {
+    maxWidth: '30rem',
+    backgroundColor: '#1E2329',
+  },
+});
+
+export const CSProp = () => {
+  return (
+    <Card cs={myStencil()}>
+      <Card.Heading cs={{color: cssVar(myStencil.vars.color)}}>The Future of Styling</Card.Heading>
+      <Card.Body cs={{color: '#fff'}}>
```

Comment:
Do we want to use raw hex values here?

Reply:
i can update to use tokens, i think i was testing!